### PR TITLE
TFinfluence update

### LINKED ISSUE #123: Add column with the supported MAVISp residue range in the output of `TFInfluence`
this follows #120

### PR DESCRIPTION
**Pull request by annamelidi**
Fix #123 .

### CODE REVIEW

File: tests/testthat/test-TFinfluence.R [hunk 1]
```diff
@@ -17,31 +19,35 @@ reference_TFresults <- data.frame(
     PMID = c(NA, NA, "18694960", "15778351"),
     tf_mutation = as.character(c(NA, NA, NA, NA)),
     stab_class = as.character(c(NA, NA, NA, NA)),
-    stringsAsFactors = FALSE # Ensure non-character columns are not treated as factors
+    stringsAsFactors = FALSE, # Ensure non-character columns are not treated as factors
+    in_MAVISp = c(FALSE,FALSE,FALSE,FALSE),
+    mutation_available = c(FALSE,FALSE,FALSE,FALSE)
```
Comment (mtiberti) [inline tests/testthat/test-TFinfluence.R]: can you move these two lines to before `stringsAsFactors`? Just for consistency

File: tests/testthat/test-TFinfluence.R [hunk 2]
```diff
@@ -7,6 +7,8 @@ data(dataTRRUST)
 data(dataMAF)
 data(dataMAVISp)
 
+dataDEGs <- DEGsmatrix
```
Comment (mtiberti) [inline tests/testthat/test-TFinfluence.R]: not sure what is the point of this

File: tests/testthat/test-TFinfluence.R [hunk 3]
```diff
@@ -17,31 +19,35 @@ reference_TFresults <- data.frame(
...
-    stringsAsFactors = FALSE # Ensure non-character columns are not treated as factors
+    stringsAsFactors = FALSE, # Ensure non-character columns are not treated as factors
+    in_MAVISp = c(FALSE,FALSE,FALSE,FALSE),
+    mutation_available = c(FALSE,FALSE,FALSE,FALSE)
 )
 
 TFresults <- TFinfluence(dataTRRUST = dataTRRUST,
             dataMAF = dataMAF,
-            dataDEGs = DEGsmatrix,
+            dataDEGs = dataDEGs,
```
Comment (mtiberti) [inline tests/testthat/test-TFinfluence.R]: not sure what is the point of this

**Review by mtiberti — CHANGES_REQUESTED** [review]

File: tests/testthat/test-TFinfluence.R (hunk 2 above)
Comment (annamelidi) [reply tests/testthat/test-TFinfluence.R]: Removed it.

**Pull request merged**